library/general/src/lib/config_file/config_file.rb

```diff
@@ -0,0 +1,29 @@
+module ConfigFile
+  class ConfigFile
```

Comment:
About naming:
- `ConfigFile` for the whole module sounds good.
- `ConfigFile::ConfigFile` -> `ConfigFile::ModelBase`
- `GrubConfig` -> `GrubModel`
- `KeyValue_Parser` -> `KeyValueParser`
- `AugeasParser` sounds good